### PR TITLE
Package distributed.0.6.0

### PR DESCRIPTION
**Pull request by essdotteedot**
### `distributed.0.6.0`
Library to provide Erlang style distributed computations. This library is inspired by Cloud Haskell
Primitive for spawning processes (in the Erlang sense) either remotely or locally, monitoring/unmonitoring spawned processes, sending, 
receiving, broadcasting messages to those processes. Unlike Erlang, the messages that are sent between processes are typed.

---
* Homepage: https://github.com/essdotteedot/distributed
* Source repo: git+https://github.com/essdotteedot/distributed.git
* Bug tracker: https://github.com/essdotteedot/distributed/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

**Comment by essdotteedot** [comment]
The revdeps test failed for "opam install -t distributed-lwt.0.1.0" and "opam install -t distributed-uwt.0.1.0". Both packages are on jbuilder and they need to be moved to dune. One of the purposes of this release is to to move the family of packages over to dune. Once distributed 0.6.0 is merged I will release dune versions of  distributed-lwt (0.2.0) and distributed-uwt (0.2.0) which will use dune.

File: packages/distributed/distributed.0.6.0/opam [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+dev-repo: "git+https://github.com/essdotteedot/distributed.git"
+bug-reports: "https://github.com/essdotteedot/distributed/issues"
+doc: "https://essdotteedot.github.io/distributed/"
+
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+
+depends: [
+  "dune" {build & >= "1.11.0"}
```
Comment (kit-ty-kate) [inline packages/distributed/distributed.0.6.0/opam]:
```suggestion
  "dune" {>= "1.11.0"}
```
The problem with the revdeps comes from this. Dune should not be tagged as a build dependency (see #14266)

Comment (essdotteedot) [reply packages/distributed/distributed.0.6.0/opam]: Thanks. I've made the suggested changes.

**Comment by kit-ty-kate** [comment]
thanks!

**Pull request merged**